packages/terra-action-header/CHANGELOG.md

```diff
@@ -10,9 +10,13 @@ Unreleased
 * Removed node_modules from .npmignore
 
 ### Changed
+<<<<<<< HEAD
```

Comment:
merge conflict

Reply:
Updated in this commit: Remove merge conflict comments